code/_onclick/item_attack.dm

```diff
@@ -8,7 +8,16 @@
 	return
 
 /obj/attackby(obj/item/I, mob/living/user, params)
-	return I.attack_obj(src, user)
+	if(unique_rename)

```

Comment:
`if(unique_rename && istype(I, /obj/item/weapon/pen))` so that it doesn't override on all objects with unique rename